### PR TITLE
fix: handle emojis properly

### LINKED ISSUE #80: improve emoji character handling
for the game "quoiFeur" we do use emoji to reply, we store all of them in an array.

We could have a function that takes an alphanumeric string and returns the emoji string.

Idea comes from the guide :
https://discordjs.guide/popular-topics/faq.html#why-do-some-emojis-behave-weirdly

### PR DESCRIPTION
**Pull request by potb**
Fixes #80

### CODE REVIEW

File: src/helpers/emojis.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+export const EMOJIS = {
```
Comment (luca-montaigut) [inline src/helpers/emojis.ts:1]: Here I prefer singular but idk if there is a real convention

**Review by luca-montaigut — DISMISSED** [review]

**Review by luca-montaigut — COMMENTED** [review]
maybe you can enforce the type or `reactWith` now that we have the EMOJIS enum ?

File: src/helpers/emojis.ts (hunk 1 above)
Comment (potb) [reply src/helpers/emojis.ts]: you're right

File: src/helpers/emojis.ts [hunk 2]
```diff
@@ -25,8 +25,10 @@ export const EMOJIS = {
...
-  muted: '🔇',
-  putLitterInItsPlace: '🚮',
-  dash: '💨',
-  wave: '👋',
+  MUTED: '🔇',
+  PUT_LITTER_IN_ITS_PLACE: '🚮',
+  DASH: '💨',
+  WAVE: '👋',
+  OK: '✅',
+  NOK: '❌',
```
Comment (luca-montaigut) [inline src/helpers/emojis.ts]: NOT_OK

**Review by luca-montaigut — APPROVED** [review]

**Review by luca-montaigut — APPROVED** [review]

**Pull request merged**